dmff/utils.py

```diff
@@ -28,3 +28,15 @@ def pair_buffer_scales(p):
             p[0] - p[1], 
             (p[0] - p[1] < 0, p[0] - p[1] >= 0), 
             (lambda x: jnp.array(1), lambda x: jnp.array(0)))
+
+
+def isinstance_jnp(*args):
```

Comment:
Have you check the jit-compatibility of this function? That is, can this function can be jitted, and would it effect the efficiency of jit? I personally would prefer to leave this check outside of the computational kernel

Reply:
I did a small test and it seems this function does not work properly when being jitted